web/src/main/resources/application.properties

```diff
@@ -23,3 +23,4 @@ oppgave.rs.uri=http://oppgave.oppgavehandtering/api/v1/oppgaver
 saf.base.url=http://saf.teamdokumenthandtering
 # Klienter (lokal)
 dokarkiv.base.url=http://dokarkiv.teamdokumenthandtering/rest/journalpostapi/v1/journalpost
+sikkerhetsnett.url=http://dokarkiv.teamdokumenthandteringrest/journalpostapi/v1/finnMottatteJournalposter
```

Comment:
Det mangler en slash her.

Reply:
Fikser den. Begynner med daglig + intervall = 1 dag for å sammenligne med doksikkerhetsnett + debug.
Kan ta lang tid før det dukker opp et ekte tilfelle ....

Reply:
Det finnes over 1000 i dev om jeg husker riktig.